eg/servo-slider.js

```diff
@@ -4,7 +4,7 @@ const board = new Board();
 board.on("ready", () => {
 
   const slider = new Sensor("A0");
-  const tilt = new Servo(9);
+  const tilt = new Servo(10);
```

Comment:
[The fritzing diagram for this example](https://github.com/rwaldron/johnny-five/blob/master/docs/servo-slider.md) uses pin 10, so this makes it consistent.